assets/section-image-banner.css

```diff
@@ -49,6 +89,42 @@
   .banner--large:not(.banner--adapt) {
     min-height: 72rem;
   }
+
+  .banner__content.banner__content--start-start,
+  .banner__content.banner__content--start-center,
+  .banner__content.banner__content--start-end {
+    align-items: flex-start;
+  }
+
+  .banner__content.banner__content--start-start,
+  .banner__content.banner__content--center-start,
+  .banner__content.banner__content--end-start {
+    justify-content: flex-start;
+  }
+
+  .banner__content.banner__content--center-start,
+  .banner__content.banner__content--center-end,
+  .banner__content.banner__content--center-center {
+    align-items: center;
+  }
+
+  .banner__content.banner__content--start-center,
+  .banner__content.banner__content--center-center,
+  .banner__content.banner__content--end-center {
+    justify-content: center;
+  }
+
+  .banner__content.banner__content--end-start,
+  .banner__content.banner__content--end-end,
+  .banner__content.banner__content--end-center {
```

Comment:
I like this idea, before I make the changes @martinamarien what do you think? 👀

Reply:
Hmm just tried it on my PR but there is an issue because the length of the value can't be more than 50 characters...

Reply:
I was looking into it further and to reduce the amount of selectors we could do it this way but it might impacts legibility: 
```css
[class*="slideshow__text-wrapper--top"] {
    align-items: flex-start;
  }

[class*="slideshow__text-wrapper"][class*="-left"] {
    justify-content: flex-start;
  }
```

Reply:
@martinamarien @ludoboludo Also I wonder if thatd be an issue if we have more than one banner section 🤔

Reply:
Update: following what we have on the designs/Ludo's slideshow PR for consistency and so it's more readable:

https://github.com/Shopify/dawn/pull/900/commits/7f0e52318bfce09daaefc8aeda9a01ec373f4a4b